File: pages/classes/SEOPage.js
class SEOPage {
  constructor(title, description, keywords = []) {
    this.title = title;
    this.description = description;
    this.keywords = keywords;
  }

  analyzeSEO = () => {
    console.log('Analyzing SEO...');
  }
}

export default SEOPage;

File: web_logic/pages/seo_page.py
from web_logic.pages.ipage import IPage


class SEOPage:

    def __init__(self,
                 title,
                 description,
                 keywords = None):
        self.title = title
        self.description = description
        self.keywords = keywords if not None else []

    def analyze_seo(self):
        pass


